src/System.Collections/tests/Performance/Perf.Dictionary.netcoreapp.cs

```diff
@@ -0,0 +1,473 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
why we still having this file? should we just update the file Perf.Dictionary.cs to work for both netcoreapp and netfx after you have introduced the extension method?